packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.CustomMenuItem.Example.tsx

```diff
@@ -1,55 +1,46 @@
 import * as React from 'react';
-import { ContextualMenuItemType } from 'office-ui-fabric-react/lib/ContextualMenu';
-import { IContextualMenuItemProps } from 'office-ui-fabric-react/lib/ContextualMenu';
+import { ContextualMenuItemType, IContextualMenuItem, IContextualMenuItemProps } from 'office-ui-fabric-react/lib/ContextualMenu';
 import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
-import './ContextualMenuExample.scss';
 
-export class ContextualMenuWithCustomMenuItemExample extends React.Component {
-  constructor(props: {}) {
-    super(props);
-    this.state = {
-      showCallout: false
-    };
-  }
+export const ContextualMenuWithCustomMenuItemExample: React.FunctionComponent = () => {
+  return (
+    <div>
+      <DefaultButton
+        text="Click for ContextualMenu"
+        menuProps={{
```

Comment:
Again being pedantic, you should probably do `useMemo` for these props. (same in other files)